apps/condo/codegen.ts

```diff
@@ -0,0 +1,47 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+
+const CodegenCommentPlugin = {
+    add: {
+        content: '// Autogenerated file by @graphql-codegen/cli\n',
```

Comment:
```suggestion
        content: [
            '// This file is autogenerated by @graphql-codegen/cli\n',
            '// Do not edit / add anything manually, since it will be overridden by codegen'
        ],
```